packages/babel-parser/src/parser/expression.ts

```diff
@@ -2010,8 +2010,11 @@ export default abstract class ExpressionParser extends LValParser {
     if (value === null) {
       if (!isTagged) {
         this.raise(Errors.InvalidEscapeSequenceTemplate, {
-          // FIXME: explain
-          at: createPositionWithColumnOffset(startLoc, 2),
+          // FIXME: Adding 1 is probably wrong.
+          at: createPositionWithColumnOffset(
+            this.state.firstInvalidTemplateEscapePos,
+            1,
```

Comment:
I noticed that all the template-related error messages have a wrong location, but I kept the bug to avoid a ton of updated fixtures. I will remove it in a separate PR.